ADOPTERS.md

```diff
@@ -7,6 +7,7 @@ alphabetical order.
 | Organization | Contact | Description of Use |
 | ------------ | ------- | ------------------ |
 | [Telia](https://www.teliacompany.com/) | [@akeed](https://github.com/akeed), [@sarahberenji](https://github.com/sarahberenji), [@tarekabouzeid](https://github.com/tarekabouzeid) | Kubeflow is used as our main ML platform for on-prem analytics. |
+| [DHL Data & Analytics](https://www.linkedin.com/company/dhl-data-analytics/) | [@juliusvonkohout](https://github.com/juliusvonkohout) | Kubeflow is used as our main ML platform. |
```

Comment:
Please can we keep this list alphabetical @juliusvonkohout ?

Reply:
Done